crates/ruff/src/rules/flake8_pytest_style/rules/fixture.rs

```diff
@@ -23,6 +23,28 @@ use super::helpers::{
     get_mark_decorators, is_pytest_fixture, is_pytest_yield_fixture, keyword_is_literal,
 };
 
+/// ## What it does
+/// Checks whether `@pytest.fixture()` without parameters has parentheses. The preferred style
+/// can be configured with the `flake8-pytest-style.fixture-parentheses` setting.
+///
+/// ## Why is this bad?
+/// Code style inconsistency.
+///
+/// ## Example
+/// ```python
+/// @pytest.fixture
+/// def my_fixture():
+///     ...
+/// ```
+///
+/// Use instead:
+/// ```python
+/// @pytest.fixture()
+/// def my_fixture():
+///     ...
+/// ```
+/// ## Options
+/// - `flake8-pytest-style.fixture-parentheses`
```

Comment:
```suggestion
/// - `flake8-pytest-style.fixture-parentheses`
///
/// ## References
/// - [API Reference: Fixtures](https://docs.pytest.org/en/latest/reference/reference.html#fixtures-api)
```

Reply:
It might be helpful to add a link to the docs (they also use the decorator without parentheses).

Reply:
Sounds good. Let's add it.